Automattic/wp-calypso
# FSE: Update Newspack to 1.7.0

#### Changes proposed in this Pull Request

* Updates newspack-blocks to 1.7.0.

#### Testing instructions



* Load posts without newspack blocks and make sure there are no scripts loaded in the front-end
* On posts that contain newspack blocks, make sure they still work as expected
